mailbox/api/src/main/java/org/apache/james/mailbox/model/UpdatedFlags.java

```diff
@@ -36,6 +38,45 @@
  */
 public class UpdatedFlags {
 
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private MessageUid uid;
+        private Flags oldFlags;
+        private Flags newFlags;
+        private Optional<Long> modSeq = Optional.absent();
+
```

Comment:
Aren't supposed to make the constructor private since there is a static method above to construct the builder